src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -186,36 +186,14 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                     {% endblock social_networks %}
 
                     {% block run_open_single %}
-                        {% with runs_dict=course.course_runs_dict %}
-                            {% with open_visible_runs=runs_dict.0|add:runs_dict.1|add:runs_dict.2|visible_on_course_page:request.toolbar.edit_mode_active %}
-                                {% if open_visible_runs|length == 0 %}
-                                    <div class="course-detail__row course-detail__runs course-detail__runs--open">
-                                        <div class="course-detail__empty">{% trans "No open course runs" %}</div>
-                                        {% is_feature_enabled 'WISHLIST' as joanie_wishlist_enabled %}
-                                        {% if is_joanie_enabled and joanie_wishlist_enabled  %}
-                                            <div
-                                                class="richie-react richie-react--course-add-to-wishlist"
-                                                data-props='{"courseCode":"{{ course.code }}"}'></div>
-                                        {% endif %}
-                                    </div>
-                                {% elif open_visible_runs|length == 1 %}
-                                    <div class="course-detail__row course-detail__runs course-detail__runs--open">
-                                        {% for run in open_visible_runs|slice:":1" %}
-                                            {% include "courses/cms/fragment_course_run.html" %}
-                                        {% endfor %}
-                                    </div>
-                                {% else %}
-                                    <div class="course-detail__row course-detail__runs course-detail__go-to-open-runs">
-                                        <p>{{ open_visible_runs|length }} {% trans "course runs are currently open for this course" %}</p>
-                                        <a class="button button--primary"
-                                            href="#courseDetailsRunsOpen"
-                                            onclick="document.getElementById(this.getAttribute('href').substring(1)).scrollIntoView({behavior: 'smooth', block: 'start'}); return false;">
-                                            {% trans "Choose now" %}
-                                        </a>
-                                    </div>
-                                {% endif %}
-                            {% endwith %}
-                        {% endwith %}
+                        <div class="richie-react richie-react--syllabus-course-runs-list" data-props='{% course_runs_list_widget_props %}'></div>
+                        {% for run in course.course_runs %}
+                            {% if run.is_visible_on_course_page or request.toolbar.edit_mode_active %}
```

Comment:
I think this is enough nope ?
```suggestion
                            {% if request.toolbar.edit_mode_active %}
```